src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -31,19 +35,51 @@ public void init() {
   }
 
   @Override
-  // Get a specific user. Return null if not found. TODO: User Validation
+  // Get a specific user. Return the currently signed in user if no userID is
+  // supplied.
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGithub = request.getParameter("githubID");
-    Key userKey = KeyFactory.createKey(PersonBuilder.ENTITY_NAME, userGithub);
-    System.out.println(userKey);
 
+    String userID =
+        request.getParameter("userID") != null
+            ? request.getParameter("userID")
+            : AuthServlet.getAuthorizedUser().getUserId();
```

Comment:
Could we make this into an if statement? I think it would be much more readable:
```suggestion
    String userID = request.getParameter("userID");
    if (userID == null) {
      userID = AuthServlet.getAuthorizedUser().getUserId();
    }
```

Also keep in mind that in your current implementation and in this suggested implementation you are at risk of a null pointer exception here if no `userID` is provided with the request and the user is not authenticated, in which case `getAuthorizedUser()` will return null.